State analysis

jwal1024 opened the pull request:
Move analysis to Jupyterbook for output of plots. Python script now generates data for inference in the Jupyterbook. Edits on the python scripts.

jwal1024 committed:
State Pivot Tables

Improvements to function for reading data from GitHub, use of new microdf groupby to simplify functions, pivot tables of state-based % changes in poverty and gini.

jwal1024 committed:
Create ipynb, edits.

pep8speaks commented:
Hello @jwal1024! Thanks for opening this PR. We checked the lines you've touched for [PEP 8](https://www.python.org/dev/peps/pep-0008) issues, and found:



* In the file [`jb/child_allowance_costs.py`](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py):

> [Line 3:55](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L3): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 19:69](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L19): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 102:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L102): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 105:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L105): [E265](https://duckduckgo.com/?q=pep8%20E265) block comment should start with '# '
> [Line 129:57](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L129): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 130:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L130): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 131:43](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L131): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 131:62](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L131): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 131:71](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L131): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 131:80](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L131): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (99 > 79 characters)
> [Line 134:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L134): [E305](https://duckduckgo.com/?q=pep8%20E305) expected 2 blank lines after class or function definition, found 1
> [Line 138:58](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L138): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 138:73](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L138): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 138:80](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L138): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (124 > 79 characters)
> [Line 152:80](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L152): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (97 > 79 characters)
> [Line 163:72](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L163): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 166:76](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L166): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 169:11](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L169): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 184:41](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L184): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 184:63](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L184): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 184:72](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L184): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 184:80](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L184): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (129 > 79 characters)
> [Line 184:82](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L184): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 184:91](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L184): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 184:106](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L184): [E231](https://duckduckgo.com/?q=pep8%20E231) missing whitespace after ','
> [Line 190:45](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L190): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 193:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L193): [E302](https://duckduckgo.com/?q=pep8%20E302) expected 2 blank lines, found 1
> [Line 195:80](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L195): [E501](https://duckduckgo.com/?q=pep8%20E501) line too long (109 > 79 characters)
> [Line 196:9](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L196): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 197:9](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L197): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 198:9](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L198): [E128](https://duckduckgo.com/?q=pep8%20E128) continuation line under-indented for visual indent
> [Line 198:25](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L198): [W291](https://duckduckgo.com/?q=pep8%20W291) trailing whitespace
> [Line 199:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L199): [E124](https://duckduckgo.com/?q=pep8%20E124) closing bracket does not match visual indentation
> [Line 201:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L201): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 202:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L202): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 202:28](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L202): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 202:30](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L202): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 203:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L203): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 203:28](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L203): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 203:30](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L203): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 204:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L204): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 204:28](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L204): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 204:30](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L204): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 205:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L205): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 205:27](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L205): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 205:29](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L205): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 206:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L206): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 206:27](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L206): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 206:29](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L206): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 207:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L207): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 207:27](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L207): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 207:29](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L207): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 210:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L210): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 210:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L210): [E266](https://duckduckgo.com/?q=pep8%20E266) too many leading '#' for block comment
> [Line 211:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L211): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 211:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L211): [E266](https://duckduckgo.com/?q=pep8%20E266) too many leading '#' for block comment
> [Line 212:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L212): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 213:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L213): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 213:14](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L213): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 213:16](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L213): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 215:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L215): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 217:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L217): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 217:14](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L217): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 217:16](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L217): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 219:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L219): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 221:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L221): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 222:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L222): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 222:25](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L222): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 222:27](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L222): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 225:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L225): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 233:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L233): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 234:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L234): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 234:13](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L234): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 234:15](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L234): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 235:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L235): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 235:17](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L235): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 235:19](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L235): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 236:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L236): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 236:25](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L236): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 236:27](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L236): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 237:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L237): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 237:19](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L237): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 237:21](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L237): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 238:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L238): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 238:19](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L238): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 238:21](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L238): [E251](https://duckduckgo.com/?q=pep8%20E251) unexpected spaces around keyword / parameter equals
> [Line 241:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L241): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 242:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L242): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 244:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L244): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 245:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L245): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 247:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L247): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 249:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L249): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented
> [Line 250:1](https://github.com/UBICenter/child-allowance/blob/f764d5b2e642fc00419d77d8b77fb81b05251128/jb/child_allowance_costs.py#L250): [E122](https://duckduckgo.com/?q=pep8%20E122) continuation line missing indentation or outdented

jwal1024 committed:
Cleanup of first sim + upload simulated dataset

Improvements to commenting, removed pp change function, more writeup in the jupyter notebook, inclusion of dataset outputed from the .py file for direction inclusion in the Jupyter book.

jwal1024 committed:
Update child_allowance_costs.py

Begins separating out the simulation code from the analysis.

To do:
Edit the simulation code
- check the total program cost calc (weighting)
- check the output

jwal1024 committed:
Completed child_allowance and CAllowance

Python file to run the simulation and Jupyterbook with pov/ineq calcs for the CPS data finalized for Ben to get started whenever. To do: clean up the 2nd python simulation code and Jupyterbook ASAP.

MaxGhenis commented:
Please add `only_build_toc_files: true` to `_config.yml` per https://jupyterbook.org/basics/build.html#config-exclude-non-toc-files

jwal1024 committed:
Toc change

jwal1024 committed:
Update _config.yml

MaxGhenis commented:
@jwal1024 looks like the notebook is broken? Second cell (cell 6): https://github.com/UBICenter/child-allowance/blob/master/jb/CAllowance.ipynb
